lib/new_relic/agent/transaction/request_attributes.rb

```diff
@@ -8,11 +8,17 @@ module NewRelic
   module Agent
     class Transaction
       class RequestAttributes
-        attr_reader :request_path, :referer, :accept, :content_length, :content_type,
-          :host, :port, :user_agent, :request_method
+        # the HTTP standard has "referrer" mispelled as "referer"
```

Comment:
lol oh no. Good comment

Reply:
I wonder how spelling bees handle this...